mvnw

```diff
@@ -0,0 +1,286 @@
+#!/bin/sh
```

Comment:
As I've said before, everyone working on this project has Maven already, so the Maven wrappers (just `mvnw` and `mvnw.cmd` in this case) are unnecessary and distracting additions.

This would be less of a concern (and also easier to address) if you were submitting your code in smaller pieces, and a "set up the project" PR included the Maven wrappers.

Reply:
I agree. I would not put this as part of our project.

Reply:
You should also remove `mvnw.cmd`